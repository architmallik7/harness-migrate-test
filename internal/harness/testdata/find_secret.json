{
    "status": "SUCCESS",
    "data": {
        "secret": {
            "type": "SecretText",
            "name": "usernamesaucelab",
            "identifier": "usernamesaucelab",
            "orgIdentifier": "default",
            "projectIdentifier": "pranavt1",
            "tags": {},
            "description": "",
            "spec": {
                "secretManagerIdentifier": "harnessSecretManager",
                "valueType": "Inline",
                "value": null,
                "additionalMetadata": null
            }
        },
        "createdAt": 1670523065771,
        "updatedAt": 1670523065771,
        "draft": false,
        "governanceMetadata": null
    },
    "metaData": null,
    "correlationId": "144e2a79-35d0-4370-b9db-b1579ef94699"
}