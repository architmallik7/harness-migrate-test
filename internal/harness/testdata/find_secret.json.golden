{
  "name": "usernamesaucelab",
  "identifier": "usernamesaucelab",
  "orgIdentifier": "default",
  "projectIdentifier": "pranavt1",
  "type": "SecretText",
  "spec": {
    "value": null,
    "valueType": "Inline",
    "secretManagerIdentifier": "harnessSecretManager"
  }
}